assets/src/edit-story/app/media/media3p/useFetchCategoriesEffect.js

```diff
@@ -29,6 +29,24 @@ import { useSnackbar } from '../../snackbar';
 import { useMedia3pApi } from './api';
 import { PROVIDERS } from './providerConfiguration';
 
+/**
+ * @typedef {Object} FetchCategoriesEffectParams
```

Comment:
Would it make sense to add these inline to the useFetchCategoriesEffect jsdoc?

Reply:
How would you do this? Do you mean by doing 
@param {Object} obj
@param {string} obj.id 

etc...?

Reply:
I just felt like it was cleaner to do this rather than to have a bunch of @params but that was just a personal choice.

I thought the @param made it seem like there were a lot of parameters in the function when theres only 1 (however that doesn't really matter when you hover over the function).

Reply:
Another way to look at it is they are named parameters, which javascript implements by providing an object.

I'll leave it up to you, feel free to use FetchCategoriesEffectParams if that's easier to reason about.